crates/ruff/src/checkers/imports.rs

```diff
@@ -25,7 +26,7 @@ pub fn check_imports(
     autofix: flags::Autofix,
     path: &Path,
     package: Option<&Path>,
-) -> Vec<Diagnostic> {
+) -> (Vec<Diagnostic>, FxHashMap<PathBuf, Vec<Import>>) {
```

Comment:
`FxHashMap<PathBuf, Vec<Import>>` is now used in many places. Can we introduce a new-type that gives it a name and abstracts a way the internal data structure

```rust
#[derive(Debug, Clone, Default)]
struct Imports {
	inner: FxHashMap<PathBuf, Vec<Import>>
}

impl Imports {
	pub fn insert(...) {}
}
```

Reply:
Sure, I'll look into doing this